### ISSUE TITLE
Impaired ability to plan

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#5852]
**Preferred term label:**
Impaired ability to plan

**Synonyms**
Impaired planning

**Definition (free text, please give PubMed ID)**
Impaired ability to anticipate future events, implement instructions or goals, and develop appropriate steps ahead of time in order to carry out a task or activity. 
Source: NCIT:C121469

**Parent term (use hpo.jax.org/app)**
HP_0030216 Inertia ?
Def is: Reduction of goal-directed behaviors linked to the impairment in frontal executive functions (planning of an action for example).

This is needed for the Kids First INCLUDE supplement.

**Comment by pnrobinson** [comment]
adding as HP_0033055
